extension/leaderelector/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+type: leader_elector
+
+status:
+  class: extension
+  stability:
+    development: [extension]
+  distributions: []
+  codeowners:
+    active: [rakesh-garimella, k15r, skhalash, hisarbalik]
```

Comment:
You'll need to update .github/CODEOWNERS to match. Typically this is done by typing:
`make gengithub`
or
```
make install-tools
.tools/githubgen -skipgithub
```
if you don't have a PAT defined to check github org membership.

Reply:
fixed using `.tools/githubgen -skipgithub`